index.d.ts

```diff
@@ -0,0 +1,70 @@
+/**
+  @param options   -  The options for renaming.
+
+  @returns         -  The new filename for the downloaded file.
+
+  @description     -  This function is used to determine the filename for each downloaded file.
+*/
+export type RenameFunction = (options: {
+	url: string;
+	index: number;
+	urls: string[];
+}) => string;
+
+/*
+  Options for the multiDownload function.
+ */
+export type Options = {
+	/**
+	  @default undefined  - (use original filenames)
+
+	  @description        - A function that returns the new filename for each downloaded file.
+    */
+	rename?: RenameFunction;
+	/**
+	  The delay time between each file download in milliseconds.
+	  @default 1000
+	 */
+	downloadInterval?: number;
+};
+
+/**
+  @param urls     - The URLs to download.
+
+  @param options  - The options for downloading.
+
+  @returns        - A promise that resolves when all files have been downloaded.
```

Comment:
Use the correct TSDoc syntax. Applies to the everything else too.

```suggestion
  @returns A promise that resolves when all files have been downloaded.
```